webroot/results/admin/persons_finish_unfinished.php

```diff
@@ -178,11 +184,14 @@ function showUnfinishedPersons () {
     #--- Html-ify name and country.
     $nameHtml = htmlEscape( $name );
     $countryIdHtml = htmlEscape( $countryId );
+    $personId = htmlEscape( $person['personId'] );
+    $competitionId = htmlEscape( $person['competitionId'] );
+    $dob = empty($person['dob']) ? 'mm/dd/yyyy' : $person['dob'];
 
     #--- Hidden field describing the case.
     $caseNr++;
-    tableRowFull( "&nbsp;<input type='hidden' name='oldNameAndCountry$caseNr' value='$nameHtml|$countryIdHtml' />" );
-    
+    tableRowFull( "&nbsp;<input type='hidden' name='oldNameAndCountry$caseNr' value='$nameHtml|$countryIdHtml|$personId|$competitionId' />" );
```

Comment:
This parameter name seems inaccurate now? Should this be `oldNameAndCountryAndPersonIdAndCompId`? ;)